### PR TITLE
Remove container-image-build job from pr.yaml workflow

### PR DESCRIPTION
**Pull request by gazarenkov**
## Description

Looks like we do not need to test building image in this flow, it's intension to test dev work.
I just commented it out in pr.yaml for the time, if it is OK - will remove this script.

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]

**Comment by gazarenkov** [comment]
> Linting and testing are also done here:
> 
> * https://github.com/janus-idp/operator/blob/main/.github/workflows/pr-container-build.yaml#L95
> * https://github.com/janus-idp/operator/blob/main/Makefile#L151
> 
> However the recent change to add gosec test should have been done for the container image build, not this golang only build.
> 
> [e345724#diff-1eb4e5fd5611777d4e597ef299a1cb5ba8050c28a2dabbd4fbc56205d69e5ddaR27](https://github.com/janus-idp/operator/commit/e34572411a88864538142e7b5ef8b53a3dd7ea0f#diff-1eb4e5fd5611777d4e597ef299a1cb5ba8050c28a2dabbd4fbc56205d69e5ddaR27)

Sure, but the purpose of this is a bit different.
As a developer I'd like to have simple and straightforward script/flow to be able to test my pull request on GutHub, it saves my time and free of charge.

File: bundle/manifests/backstage-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -206,7 +206,7 @@ spec:
                   value: quay.io/fedora/postgresql-15:latest
                 - name: RELATED_IMAGE_backstage
                   value: quay.io/janus-idp/backstage-showcase:next
-                image: quay.io/janus-idp/operator:0.0.1
+                image: gazarenkov/backstage-operator:latest
```
Comment (nickboldt) [inline bundle/manifests/backstage-operator.clusterserviceversion.yaml:209]: this seems like an accidentally pushed change.

Comment (gazarenkov) [reply bundle/manifests/backstage-operator.clusterserviceversion.yaml:209]: yes it is

Comment (gazarenkov) [reply bundle/manifests/backstage-operator.clusterserviceversion.yaml]: fixed

**Review by nickboldt — APPROVED** [review]
/lgtm

**Pull request closed without merging**